### PR TITLE
Cloud 103 increase pvc

### PR DESCRIPTION
**Pull request by delgod**

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster-operator?pullRequest=119) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster-operator?pullRequest=119) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/percona-xtradb-cluster-operator?pullRequest=119) it.</sub>

File: install/scaling.md [hunk 1]
```diff
@@ -12,3 +12,34 @@ Size of the cluster is controlled by a `size` key in the Custom Resource options
 In this example we have changed the size of the Percona XtraDB Cluster from `3`, which is a minimum recommended value, to `5` nodes.
 
 **Note:** *Using ```kubectl scale StatefulSet_name``` command to rescale Percona XtraDB Cluster is not recommended, as it makes `size` configuration option out of sync, and the next config change may result in reverting the previous number of nodes.*
+
```
Comment (tduzan) [inline install/scaling.md]: I think this should have a subheading, as should the preceding selection to separate the acts of scaling instance count in the cluster and scaling volume size for existing instances in the cluster.

File: scaling.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+Scale Percona XtraDB Cluster on Kubernetes and OpenShift
+----------------------------------------------------------
+
```
Comment (tduzan) [inline scaling.md]: There should be a separate subheading for this section

File: install/scaling.md [hunk 3]
```diff
@@ -12,3 +12,56 @@ Size of the cluster is controlled by a `size` key in the Custom Resource options
...
+### Increase the Persistent Volume Claim size
+
+Kubernetes manages storage with a PersistentVolume (PV), a segment of storage supplied by the administrator, and a PersistentVolumeClaim (PVC), a request for storage from a user. In Kubernetes v1.11 the feature was added to allow a user to increase the size of an existing PVC object. The user cannot shrink the size of an existing PVC object. Certain volume types support, be default, expanding PVCs (details about PVCs and support volume types can be found in [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims))
+
+The following are the steps to increase the size:
+0. Extract and backup the yaml file for the cluster
```
Comment (fiowro) [inline install/scaling.md]:
```suggestion

0. Extract and backup the yaml file for the cluster
```

File: install/scaling.md [hunk 4]
```diff
@@ -12,3 +12,56 @@ Size of the cluster is controlled by a `size` key in the Custom Resource options
 In this example we have changed the size of the Percona XtraDB Cluster from `3`, which is a minimum recommended value, to `5` nodes.
 
 **Note:** *Using ```kubectl scale StatefulSet_name``` command to rescale Percona XtraDB Cluster is not recommended, as it makes `size` configuration option out of sync, and the next config change may result in reverting the previous number of nodes.*
+
+### Increase the Persistent Volume Claim size
+
+Kubernetes manages storage with a PersistentVolume (PV), a segment of storage supplied by the administrator, and a PersistentVolumeClaim (PVC), a request for storage from a user. In Kubernetes v1.11 the feature was added to allow a user to increase the size of an existing PVC object. The user cannot shrink the size of an existing PVC object. Certain volume types support, be default, expanding PVCs (details about PVCs and support volume types can be found in [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims))
```
Comment (fiowro) [inline install/scaling.md]:
```suggestion
Kubernetes manages storage with a PersistentVolume (PV), a segment of storage supplied by the administrator, and a PersistentVolumeClaim (PVC), a request for storage from a user. In Kubernetes v1.11 the feature was added to allow a user to increase the size of an existing PVC object. The user cannot shrink the size of an existing PVC object. Certain volume types support, by default, expanding PVCs (details about PVCs and support volume types can be found in [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims))
```

**Pull request merged**